themes/stanford_basic/algolia-search/src/styled-components.tsx

```diff
@@ -113,20 +113,30 @@ export const SearchForm = styled.form`
       display: flex;
       flex-direction: row;
       padding: 1.2rem 2rem 1.2rem;
-      background-color: #f4f4f4;
+      background-color: #fff;
 
       &:hover, &:focus {
-        background: transparent;
+        background-color: #f4f4f4;
         color: #2E2D29;
-        text-decoration: underline;
         box-shadow: none;
         outline: none;
 
-        i {
+        span {
+          text-decoration: underline;
+        }
+      }
+
+      &:focus-visible, &:focus {
```

Comment:
you don't need both. that's redundant. `:focus` is when a keyboard or a mouse focuses. `:focus-visible` is when a keyboard focuses. So use `focus-visible` and `hover` separately